homeassistant/components/airthings_ble/sensor.py

```diff
@@ -162,10 +162,11 @@ def __init__(
         super().__init__(coordinator)
         self.entity_description = entity_description
 
-        name = f"{airthings_device.name} {airthings_device.identifier}"
+        name = f"{airthings_device.name}"
+        if airthings_device.identifier != "":
+            name += f" ({airthings_device.identifier})"
```

Comment:
name is already typed to be a string


https://github.com/home-assistant/core/pull/97905/files#r1285284428